src/js/components/dashboard/visualizations/DashboardTable.jsx

```diff
@@ -0,0 +1,144 @@
+/**
+ * DashboardTable.jsx
+ * Created by Alisa Burdeyny 11/13/19
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEqual, uniqueId } from 'lodash';
+
+import DashboardTableHeader from 'components/dashboard/visualizations/DashboardTableHeader';
+import DashboardTableRow from 'components/dashboard/visualizations/DashboardTableRow';
+
+const propTypes = {
+    results: PropTypes.array,
+    inFlight: PropTypes.bool
+};
+
+const defaultProps = {
+    results: [],
+    inFlight: false
+};
+
+const tableHeaders = [
+    {
+        text: 'File',
+        class: 'dashboard-table__file-column'
+    },
+    {
+        text: 'Fiscal Year/ Quarter',
+        class: 'dashboard-table__fyq-column'
+    },
+    {
+        text: 'Warning Rule',
+        class: 'dashboard-table__label-column'
+    },
+    {
+        text: 'Number of Instances',
+        class: 'dashboard-table__instances-column'
+    },
+    {
+        text: 'Rule Description',
+        class: null
+    }
+];
+
+export default class DashboardTable extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            parsedData: []
+        };
+
+        this.buildRows = this.buildRows.bind(this);
+    }
+
+    componentDidMount() {
+        this.buildRows();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (!isEqual(prevProps.results, this.props.results)) {
+            this.buildRows();
+        }
+    }
+
+    buildRows() {
+        const rows = [];
+        this.props.results.forEach((item) => {
+            rows.push(this.formatRow(item));
+        });
+
+        this.setState({
+            parsedData: rows
+        });
+    }
+
+    formatRow(item) {
```

Comment:
Could we move the data formatting to the container? This is also a place where the USAS data model pattern would make sense. (e.g. model for table rows on the recipient landing page: https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/models/v2/recipient/BaseRecipientLandingRow.js) We can discuss more in depth offline!